practice-areas/project-management/project-management-checklists.md

```diff
@@ -6,18 +6,19 @@ The Project Management practice area suggests leveraging the following checklist
 
 Wrap up the past month:
 
-1. Complete monthly contractual deliverables (monthly reporting, etc.)
-2. Run the "plan vs actual" report for the last month and in order to verify reporting accuracy
+1. Complete monthly contractual deliverables (monthly reporting, etc.), making sure they are in the shared folder
+2. Run the "plan vs actual" report for the last month and in order to verify reporting accuracy [See Unanet instructions](https://guidebook.civicactions.com/en/latest/practice-areas/project-management/project-management-checklists/)
```

Comment:
@hhartingrex it looks like you are linking the same page for the unanet instructions? Should it go into a different page and also can the link be relative like `pm-unanet-tasks.md`?

Reply:
@dmundra Thank you! Didn't know about the relative trick. Fixing now!